src/main/java/com/lpvs/LicensePreValidationService.java

```diff
@@ -56,9 +58,11 @@ public static void main(String[] args) {
             ApplicationContext applicationContext =
                     SpringApplication.run(LicensePreValidationService.class, args);
             exitHandler = applicationContext.getBean(LPVSExitHandler.class);
-        } catch (IllegalArgumentException e) {
-            System.err.println("An IllegalArgumentException occurred: " + e.getMessage());
+        } catch (IllegalArgumentException ex1) {
+            log.error("An IllegalArgumentException occurred: " + ex1.getMessage());
             System.exit(1);
+        } catch (Exception ex2) {
+            log.info("LPVS application is being closed.");
```

Comment:
Can we use the same variable name?
```suggestion
        } catch (IllegalArgumentException e) {
            log.error("An IllegalArgumentException occurred: " + e.getMessage());
            System.exit(1);
        } catch (Exception e) {
            log.info("LPVS application is being closed.");
```